uber/templates/emails/index.html

```diff
@@ -4,6 +4,11 @@
 
 {% pages page count %}
 
+<form action="index">
+    <input type="email" placeholder="Search by email..." name="search_text" value="{{ search_text }}">
```

Comment:
Let's say "email address" in the placeholder text, since uh... everything is an email in this page. :P

Also, for convention's sake, this should go above the {% pages page count %} (like it works on the Attendees page).